From 444600e3210dedbf480b6338c3d352a441e1713a Mon Sep 17 00:00:00 2001
From: Carlos Munoz <cmunoz@caviumnetworks.com>
Date: Thu, 5 Feb 2015 11:09:41 -0800
Subject: [PATCH 062/184] MIPS: OCTEON: Changed verbose info printk to debug
 printk.

Source: Cavium Networks, Inc.
MR: 14117
Type: Integration
Disposition: Merged from Octeon Tree
ChangeID: dcd41bac07feb75ecce87a7ea1f3a7542e6036e0
Description:

Signed-off-by: Carlos Munoz <cmunoz@caviumnetworks.com>
Signed-off-by: Corey Minyard <cminyard@mvista.com>
[Original patch taken from octeon-linux-kernel-patches-SDK-3.1.2-release]
Signed-off-by: Yanjiang Jin <yanjiang.jin@windriver.com>
---
 arch/mips/cavium-octeon/octeon-irq.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/mips/cavium-octeon/octeon-irq.c b/arch/mips/cavium-octeon/octeon-irq.c
index 1cb2a3a..e17dd0a 100644
--- a/arch/mips/cavium-octeon/octeon-irq.c
+++ b/arch/mips/cavium-octeon/octeon-irq.c
@@ -2778,7 +2778,7 @@ int octeon_ciu3_errbits_enable_intsn(int node, int intsn)
 	if (isc_ctl.s.en) {
 		union cvmx_ciu3_iscx_w1c isc_w1c;
 		u64 isc_w1c_addr = ciu3_addr + CIU3_ISC_W1C(intsn);
-		pr_info("Already enabled intsn: 0x%x\n", intsn);
+		pr_debug("Already enabled intsn: 0x%x\n", intsn);
 		isc_w1c.u64 = 0;
 		isc_w1c.s.en = 1;
 		cvmx_write_csr(isc_w1c_addr, isc_w1c.u64);
-- 
1.9.1

